src/Helpers/Form/Utils.php

```diff
@@ -242,4 +242,22 @@ public static function canDisableDonationNowButton()
     {
         return ! empty($_GET['giveDisableDonateNowButton']);
     }
+
+    /**
+     * @unreleased
+     */
+    public static function isV3Form(int $formId): bool
+    {
+        return (bool)give()->form_meta->get_meta($formId, 'formBuilderSettings', true);
+    }
+
+    /**
+     * @unreleased
+     */
+    public static function isFormBuilderRequest(): bool
+    {
+        return (isset($_POST['blocks']) && strpos($_POST['blocks'],
```

Comment:
@ravinderk Before creating this function, I tried to use [this const](https://github.com/WordPress/WordPress/blob/8e41746cb11271d063608a63e3f6091a8685e677/wp-includes/rest-api.php#L141) to check if it was a rest request, but it always returned false because the service provider is loaded before the WP rest API, and at this moment just the `$_POST` is available to be checked. 

The `$GLOBALS['wp']->query_vars['rest_route'];` is also not available at this point as you can see in the following video: https://www.loom.com/share/b4278e877d404f9faa2e4209ec6f2c3b?sid=4c5cb68e-e0f5-4bc9-90ce-b056970e56ff

Reply:
@glaubersilva In this case, I will suggest running this code on `admin_init`. Can you check?

Reply:
@ravinderk  The `loadServiceProviders()` method currently is triggered on the `plugins_loaded` hook. 

So what you are suggesting is to create a new method similar to this one to be triggered on the `admin_init` hook and load just the `TrackingServiceProvider` class, right?  

If yes, I tried it by creating the `loadAdminServiceProviders()` method hooked on the `admin_init` hook, and didn't work.

Reply:
@ravinderk I found a solution using the `rest_api_init` hook. Resolve in a1dd3f4c610923016d86887badca5d22299cfe08 and 968053fc82b51233ed14ce00762ff471c23a4f4b